### PR TITLE
[Windows] Support of JavaScript dialogs.

### PR DESCRIPTION
**Pull request by darktears**
Add a JavaScript dialog class which can be implemented in
future platforms like Linux or Mac, where one would just
provide the _platform to put the implementation.

Dialogs probably can be made nicer looking like in Chrome.

BUG=XWALK-5594

### CODE REVIEW

**Comment by darktears** [comment]
@pozdnyakov some review love...

File: runtime/browser/runtime.cc [hunk 1]
```diff
@@ -224,7 +225,10 @@ void Runtime::DidNavigateMainFramePostCommit(
 
 content::JavaScriptDialogManager* Runtime::GetJavaScriptDialogManager(
     content::WebContents* web_contents) {
-  return NULL;
+  if (!javascript_dialog_manager_.get()) {
```
Comment (pozdnyakov) [inline runtime/browser/runtime.cc]: get() is not needed

File: runtime/browser/runtime_javascript_dialog.h [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      const content::JavaScriptDialogManager::DialogClosedCallback& callback);
+  ~RuntimeJavaScriptDialog();
+
+  void Cancel();
+
+ private:
+  RuntimeJavaScriptDialogManager* manager_;
+  RuntimeJavaScriptDialogManager::DialogClosedCallback callback_;
+
+#if defined(OS_WIN)
```
Comment (pozdnyakov) [inline runtime/browser/runtime_javascript_dialog.h]: better make a hierarchy, RuntimeJavaScriptDialogWin, and a factory method

**Comment by pozdnyakov** [comment]
lgtm

**Pull request merged**